ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/tools/ExposeAllLogsTest.java

```diff
@@ -47,19 +47,18 @@
 import com.arjuna.ats.internal.arjuna.common.UidHelper;
 import com.arjuna.common.internal.util.propertyservice.BeanPopulator;
 
+
```

Comment:
Are two blank lines after the import section deliberate?

Reply:
Not sure, will change it to one line.